docs/.vuepress/config.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
         }
       ],
       smallprint:
-          'The development of the Tendermint project is led primarily by Tendermint Inc., the for-profit entity which also maintains this website. Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit.',
+          'The development of the Tendermint project is led primarily by [Interhcain GmbH](https://interchain.berlin/). Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit.',
```

Comment:
```suggestion
          'The development of the Tendermint project is led primarily by [Interhcain GmbH](https://interchain.berlin/). Funding for this development comes primarily from the Interchain Foundation, a Swiss non-profit. The Tendermint trademark is owned by Tendermint Inc.',
```

I'd like to hear @nylira's thoughts on this :+1:

Reply:
I like this. @nylira, what do you think?